gh-126745: Fixup Docu time.rst (#126744)
python/cpython: Doc/library/time.rst

   Use :func:`monotonic_ns` to avoid the precision loss caused by the
   :class:`float` type.

   .. impl-detail::

      On CPython, use the same clock as :func:`time.perf_counter` and is a

>       monotonic clock, i.e. a clock that cannot go backwards.

Reviewer: This makes more sense under _perf_counter_ were it was. All _monotonic_ function should always be monotonic, by definition.

   .. versionchanged:: 3.10
      On macOS, the function is now system-wide.

   .. versionchanged:: 3.13

>       Use the same clock as :func:`time.perf_counter`. On Windows, :func:`monotonic`

Reviewer: ~Do you mean to say this only applies to Windows? Otherwise, we don’t really need two separate pairs of functions.~

Sorry, looking at Victor’s commit 846ad5a26ac0, I see these functions do indeed now all share the same implementation, regardless of platform.

>       Use the same clock as :func:`time.perf_counter`. On Windows, :func:`monotonic`

Reviewer: ```suggestion
      The function now uses the same clock as :func:`time.perf_counter`. On Windows, :func:`monotonic`
```

>       Use the same clock as :func:`time.perf_counter`. On Windows, :func:`monotonic`
>       now call ``QueryPerformanceCounter()`` which has a resolution of 1 microsecond,

Reviewer: ```suggestion
      Use the same clock as :func:`time.perf_counter`. On Windows, :func:`monotonic`
      now calls ``QueryPerformanceCounter()`` which has a resolution of 1 microsecond,
```

   Clock:

   * On Windows, call ``GetSystemTimePreciseAsFileTime()``.
   * Call ``clock_gettime(CLOCK_REALTIME)`` if available.
   * Otherwise, call ``gettimeofday()``.

   Use :func:`time_ns` to avoid the precision loss caused by the :class:`float`
   type.

>   .. versionchanged:: 3.13

Reviewer: ```suggestion
   .. versionchanged:: 3.13
```

Suspect the mixed indentation is screwing it up
